README.md

```diff
@@ -160,23 +160,41 @@ Default way of authenticating is with the client credentials
 ```typescript
 const connection = await firebolt.connect({
   auth: {
-    client_id: 'b1c4918c-e07e-4ab2-868b-9ae84f208d26';
-    client_secret: 'secret';
+    client_id: 'b1c4918c-e07e-4ab2-868b-9ae84f208d26',
+    client_secret: 'secret',
   },
   engineName: 'engine_name',
   account: 'account_name',
   database: 'database',
 });
 ```
+
+<a id="token-caching"></a>
+#### Token caching
+Driver implements a caching mechanism for access tokens. If you are using the same client id/secret for multiple connections, the driver will cache the access token and reuse it for subsequent connections.
+This behavior can be disabled by setting `useCache` to `false` in the connection options.
+```typescript
+const connection = await firebolt.connect({
+  auth: {
+    client_id: 'b1c4918c-e07e-4ab2-868b-9ae84f208d26',
+    client_secret: 'secret',
+  },
+  engineName: 'engine_name',
+  account: 'account_name',
+  database: 'database',
+  useCache: false
+});
+```
+
+```typescript
 <a id="test-connection"></a>
```

Comment:
Let's add a space after line 189 otherwise the next tag breaks. At least in preview.